Makefile

```diff
@@ -30,7 +30,7 @@ test: install build-examples .bin/ory
 	./src/scripts/test.sh
 
 .bin/ory: Makefile
-	bash <(curl https://raw.githubusercontent.com/ory/meta/master/install.sh) -d -b .bin ory v0.1.33
+	bash <(curl https://raw.githubusercontent.com/ory/meta/master/install.sh) -d -b .bin ory v0.1.44
```

Comment:
since this is bumped to the latest I had to send all output to  /dev/null for the ory proxy / ory tunnel since this caused an io error and the ci would stop the run.